cmd/server-main.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"github.com/nic758/bdtp-golang/bdtp"
+	"github.com/nic758/bdtp-golang/cli"
+)
+
+var DefaultPort = "4444"
+
+var srvFlags = []cli.Flag{
```

Comment:
The changing of ports doesn't seem to be functioning properly, at least on my windows machine

![Capture](https://user-images.githubusercontent.com/38961332/192335051-b8b7e5fd-fd3b-44a3-b741-90b8dcd5dae3.PNG)

Reply:
Sur mon windows non plus